### PR TITLE
improved username/password promt messages for remote

### PR DESCRIPTION
**Pull request by devidask27**
Screenshot 2021-02-26 at 12 24 37 PM

### CODE REVIEW

File: cmd/prepNode.go [hunk 1]
```diff
@@ -84,7 +84,7 @@ func checkAndValidateRemote() bool {
 		if ip != "localhost" && ip != "127.0.0.1" && ip != "::1" {
 			// lets create a remote executor, but before that check if we got user and either of password or ssh-key
 			if user == "" {
-				fmt.Printf("Enter Username: ")
+				fmt.Printf("Enter remote machines username: ")
```
Comment (anupbarve) [inline cmd/prepNode.go:87]: Enter username for remote host :

File: cmd/prepNode.go [hunk 2]
```diff
@@ -97,7 +97,7 @@ func checkAndValidateRemote() bool {
 				fmt.Scanf("%d", &choice)
 				switch choice {
 					case 1:
-						fmt.Printf("Enter Password: ")
+						fmt.Printf("Enter remote machines password: ")
```
Comment (anupbarve) [inline cmd/prepNode.go:100]: Enter password for remote host :

File: cmd/prepNode.go [hunk 3]
```diff
@@ -97,7 +97,7 @@ func checkAndValidateRemote() bool {
...
 						passwordBytes, _ := terminal.ReadPassword(0)
 						password = string(passwordBytes)
 					case 2:
```
Comment (anupbarve) [inline cmd/prepNode.go:103]: Enter private sshKey:

**Review by anupbarve — CHANGES_REQUESTED** [review]

**Comment by devidask27** [comment]
TC build link
http://teamcity.platform9.horse:8111/buildConfiguration/Pf9project_Platform9Components_Pf9ctl/1449475

**Review by anupbarve — APPROVED** [review]

**Pull request merged**